group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -355,7 +370,7 @@ GroupMetadataManager build() {
     private final int consumerGroupHeartbeatIntervalMs;
```

Comment:
The comment should probably read "The default heartbeat interval for consumer groups."

Reply:
Done